coffee-backend/src/main/kotlin/chat/letscoffee/notification/NotificationController.kt

```diff
@@ -16,29 +16,55 @@ import org.springframework.web.bind.annotation.RestController
 @RequestMapping("/notifications")
 class NotificationController(
     private val notificationService: NotificationService,
+    private val notificationSubscriptionRepository: NotificationSubscriptionRepository,
```

Comment:
Considering the other refactoring that has been done in these branches, why is this controller given direct access to the repository. Should move Should ask the service for this?

Reply:
Agreed. Didn't want to touch all the services/repos at the same time. Will refactor in later PR :+1: